js/js.js

```diff
@@ -1,8 +1,125 @@
-var list = document.getElementById('Lista');
-var add = document.getElementById('addElem');
-var count = document.getElementsByTagName('li').length;
-
-add.addEventListener('click', function() {
-	list.innerHTML += '<li>item ' + count + '</li>'
-	count++;
-});
+//---------------VARIABLES---------------//
+
+var newGameBtn = document.getElementById('js-newGameButton'),
+    newGameElem = document.getElementById('js-newGameElement'),
+    pickElem = document.getElementById('js-playerPickElement'),
+    resultsElem = document.getElementById('js-resultsTableElement');
+
+var pickRock = document.getElementById('js-playerPick_rock'),
+    pickPaper = document.getElementById('js-playerPick_paper'),
+    pickScissors = document.getElementById('js-playerPick_scissors');
+
+var playerPickElem = document.getElementById('js-playerPick'),
+    computerPickElem = document.getElementById('js-computerPick'),
+    playerResultElem = document.getElementById('js-playerResult'),
+    computerResultElem = document.getElementById('js-computerResult');
+
+var playerPointsElem = document.getElementById('js-playerPoints'),
+    playerNameElem = document.getElementById('js-playerName'),
+    computerPointsElem = document.getElementById('js-computerPoints');
+
+var gameState = 'notStarted',  //started // ended
+    player = {
+        name: '',
+        score: 0
+    },
+    computer = {
+        score: 0
+    };
+
+//---------------FUNCTIONS---------------//
+
+function setGameElements() {
+  switch(gameState) {
+    case 'started':
+        newGameElem.style.display = 'none';
+        pickElem.style.display = 'block';
+        resultsElem.style.display = 'block';
+      break;
+    case 'ended':
+        newGameBtn.innerText = 'Jeszcze raz';
+    case 'notStarted':
+    default:
+        newGameElem.style.display = 'block';
+        pickElem.style.display = 'none';
+        resultsElem.style.display = 'none';
+  }
+}
+
+function getComputerPick() {
+    var possiblePicks = ['rock', 'paper', 'scissors'];
+    return possiblePicks[Math.floor(Math.random()*3)];
+}
+
+function setGamePoints() {
+    playerPointsElem.innerHTML = player.score;
+    computerPointsElem.innerHTML = computer.score;
+}
+
+function setGameEnd() {
+	if (player.score = 10) {
+		gameState = 'ended';
+	}
+	else if (computer.score = 10) {
+		gameState = 'ended';
+	}
+}
```

Comment:
Musiałem wgrać starszą wersje bo u mnie już działało. Co do Twojej propozycji byłoby  zostawiam w takie wersji:

function updateGameState() {
	if (player.score === 10) {
		alert('10 punktów zdobyte! Wygrał gracz: ' + player.name);
		gameState = 'ended';
	}
	else if (computer.score === 10) {
		alert('10 punktów zdobyte! Wygrał komputer');	
		gameState = 'ended';
	}
}

Zastanawiam się jakbym miał zrobić to na zasadzie zwracania (return) jeśli chciałbym użyć alert-a który musi mi sprawdzić który gracz ma 10 punktów.